modules/twinkleprotect.js

```diff
@@ -1405,7 +1402,7 @@ Twinkle.protect.callbacks = {
 		var text = rppPage.getPageText();
 		var statusElement = rppPage.getStatusElement();
 
-		var rppRe = new RegExp( '===\\s*(\\[\\[)?\s*:?\s*' + RegExp.escape( Morebits.pageNameNorm, true ) + '\s*(\\]\\])?\\s*===', 'm' );
+		var rppRe = new RegExp( '===\\s*(\\[\\[)?\\s*:?\\s*' + RegExp.escape( Morebits.pageNameNorm, true ) + '\\s*(\\]\\])?\\s*===', 'm' );
```

Comment:
one more \\s

Reply:
I think this one is okay; is there a specific `\s` you're referring to?

Reply:
there's a \\s at the end of the regex as well

Reply:
I read the diff backwards, I though you removed escapes :|